### DISCUSSION TITLE
Custom index.html

### DISCUSSION THREAD

**Discussion by krthr** [nuxt/nuxt#17392] [Questions]
I can't find any documentation on how to modify the `index.html` :(

I need to add a class to the `html` and `body` elements.

If anyone has any idea how to do it I appreciate the help. Thanks!

**Comment by maxim1maslov** [comment]
I can confirm, I have the same problem. I have 3 scripts that I need to add, one to the top of the head, one to the end of the head, and one to the end of the body of the html.

I thought Nuxt 3 supported the app.html that was available in Nuxt.js, but apparently I am mistaken. Havent found a solution yet.

**Comment by pi0** [comment]
Hi. I've create and issue to follow up this future addition: https://github.com/nuxt/framework/issues/5565

We probably are not adding custom HTML support but an easier config interface you can use to inject classes to HTML and Body elements. Feedback is more than welcome!

**Comment by Flanker72** [comment]
You don't need to modify any html explicitely. Nuxt have a number of helpers to manipulate the generated content.

To manipulate Html/Body/Head tags you may use these approaches:
1. Place Html, Body or/and Head [meta components](https://v3.nuxtjs.org/guide/features/head-management#meta-components) and pass the needed params to them.
2. Use `app.head` [config section](https://v3.nuxtjs.org/api/configuration/nuxt.config#head) in `nuxt.config.ts` to provide htmlAttrs/bodyAttrs there.
3. Use [useHead approach](https://v3.nuxtjs.org/guide/features/head-management#usehead-composable).
